### PR TITLE
CDI resource is not Namespace resource

### PR DESCRIPTION
**Pull request by myakove**
##### Short description:
CDI resource is not Namespace resource and should inherit from Resource.

##### Release notes:

```release-note
None
```

### CODE REVIEW

File: ocp_resources/cdi.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 
-class CDI(NamespacedResource):
+class CDI(Resource):
     """
     CDI object.
     """
 
-    api_group = NamespacedResource.ApiGroup.CDI_KUBEVIRT_IO
+    api_group = Resource.ApiGroup.CDI_KUBEVIRT_IO
```
Comment (sarahbx) [inline ocp_resources/cdi.py:9]: This should be updated across the wrapper, there are additional places this `ApiGroup` is referenced as a `NamespacedResource`

**Review by sarahbx — CHANGES_REQUESTED** [review]

File: ocp_resources/cdi.py (hunk 1 above)
Comment (myakove) [reply ocp_resources/cdi.py:9]: @sarahbx what do you mean?
each resource call his parent.

Comment (sarahbx) [reply ocp_resources/cdi.py:9]: It does due to inheritance, but this is confusing for cluster scoped resources, using both to refer to a non-namespace scoped resource.

**Comment by myakove** [comment]
```
cdi = list(CDI.get(dyn_client=admin_client))
[<ocp_resources.cdi.CDI object at 0x7f21c03f4160>]
cdi[0].name
'cdi-kubevirt-hyperconverged'
```

**Comment by ibesso-rh** [comment]
/approve
ran tests that failed before.

**Pull request merged**